cfme/base/ui.py

```diff
@@ -539,6 +539,7 @@ def step(self):
         self.view.rolesbyservers.select()
 
 
+# this destination is absent in 5.7
```

Comment:
We've got a few classes that throw an exception based on version, could do that here. think we used DestinationNotFound and OptionNotAvailable depending on the situation.